docs/manual/inference.tex

```diff
@@ -164,7 +164,15 @@
   or Maven buildfile. The default is the current working directory.
 
 \item[-t TIMEOUT]
-  The timeout for running the checker, in seconds
+  The timeout for running the checker, in seconds.
+
+\item[-c COMPILE\_CMD]
```

Comment:
I find "compile_cmd" and "the command to use" a bit misleading.
It's actually a build system target (plus additional command-line arguments to the build system? I'm not sure, but I recall the example you gave to Narges including those rather than putting them in `-b`.)
I think the naming add documentation should be clarified.

Also, please add a concrete example of what a user might pass on the command line, to further clarify.
This might help with the text "depending on the setup of the target project", which is currently vague and doesn't give actionable advice.

Reply:
I added an example. I still find it somewhat unsatisfying, but I'm not sure that it's possible to give advice that's broadly applicable here, since what the user will need to do varies project-by-project, and these options exist to allow that kind of customization. Let me know if you have any better ideas than my latest set of changes.